src/components/molecules/card/CardFooter.tsx

```diff
@@ -20,16 +21,26 @@ const useStyles = makeStyles({
     justifyContent: 'flex-end',
   },
   comment: {
-    paddingLeft: '1rem',
+    display: 'block',
   },
 });
 
 const CardFooter: React.FC<IProps> = ({ dateComment }) => {
   const classes = useStyles();
+  const store = useStore();
+  const [xPosition, setXPosition] = useState(300);
+  useEffect(() => {
+    setXPosition(store.currentLanguageRouteString === '/en' ? 0 : 300);
+  }, [store.currentLanguageRouteString]);
```

Comment:
```suggestion
const xPosition = store.currentLanguageRouteString === '/en' ? 0 : 300
```

Reply:
I changed the file to use only html-to-canvas and there is no need for SVG tag anymore